indico/modules/events/registration/templates/management/_reglist.html

```diff
@@ -124,7 +124,13 @@
                                 {% endfor %}
                                 {% for item in dynamic_columns %}
                                     {% set search_value = data[item.id].search_data if item.id in data else '' %}
-                                    {% if item.id in data and data[item.id].field_data.field.input_type == 'checkbox' %}
+                                    {% if item.id in data and data[item.id].field_data.field.is_purged %}
+                                        <td class="i-table center">
```

Comment:
I don't think changing the alignment here looks good. Maybe best to keep the default placement (ie left aligned)?

Reply:
Removed the center alignment. Looks like this now.
![image](https://github.com/user-attachments/assets/34d2eda5-388b-4878-ab94-31dc72e24606)